sale-service/src/main/java/com/sap/refapps/espm/controller/SalesOrderController.java

```diff
@@ -128,9 +128,9 @@ public ResponseEntity<String> updateSalesOrder(@PathVariable("salesOrderId") fin
 					logger.info("get response value:", response.getStatusCodeValue());
 					String responseBody = response.getBody();
 					logger.info("response body for success", responseBody);
-					if (response.getStatusCodeValue() == 200) {
+					if (response.getStatusCode().value() == 200) {

```

Comment:
can use   HttpStatus.OK.value(); instead of hardcoding values

Reply:
Done ..Thanks